packages/mobile/src/screens/chat-screen/ChatBlastAudienceDisplay.tsx

```diff
@@ -39,11 +39,13 @@ export const ChatBlastAudienceDisplay = (
       <Flex row gap='xs' alignItems='center'>
         <Text size='l'>{messages.description}</Text>
       </Flex>
-      <ProfilePictureList
-        users={users as User[]}
-        totalUserCount={audienceCount ?? 0}
-        limit={USER_LIST_LIMIT}
-      />
+      {users.length ? (
+        <ProfilePictureList
+          users={users as User[]}
```

Comment:
users is `[]` whenever the audius-query hooks that get the data haven't returned yet

E.g. we return `[]` here if audience is followers and `followers` data is undefined
```
let users: UserMetadata[] = []
switch (chat.audience) {
  case ChatBlastAudience.FOLLOWERS:
    users = followers ?? []
    break
  case ChatBlastAudience.TIPPERS:
    users = supporters ?? []
    break
  case ChatBlastAudience.CUSTOMERS:
    users = purchasers ?? []
    break
  case ChatBlastAudience.REMIXERS:
    users = remixers ?? []
    break
}
```

Reply:
yeah.. i don't understand how that leads to the error tho. `[].map` should just not do anything right?
<img width="289" alt="Screenshot 2024-09-23 at 5 53 26 PM" src="https://github.com/user-attachments/assets/d8e78414-dfae-4619-92d3-64413570ce61">

Reply:
crash was that `user_id` was undefined here: https://github.com/AudiusProject/audius-protocol/blob/d02c16c1494e18f41bd69739702c301ccdbb9b1f/packages/mobile/src/screens/notifications-screen/Notification/NotificationProfilePictureList.tsx#L126

Reply:
huh... wtf

I have no idea